photon-client/src/components/pipeline/3D/MiniMap.vue

```diff
@@ -111,21 +114,35 @@ export default {
       }
       if(this.cubes.length > 0)
         this.scene.add(...this.cubes);
-    }
+    },
+    onWindowResize() {
+      var container = document.getElementById("MapContainer")
+      if(container){
+        this.canvas.width = container.clientWidth * 0.95;
+        this.canvas.height = container.clientWidth * 0.85;
+        this.camera.aspect = this.canvas.width / this.canvas.height;
+        this.camera.updateProjectionMatrix();
+        this.renderer.setSize( this.canvas.width, this.canvas.height );
```

Comment:
Does updateProjectionMatrix need to happen before or after set size? Just curious